### ISSUE TITLE
Accessibility Fixes for Disabled/Enabled Search Button

### ISSUE THREAD

**Issue by mjoyce91** [18F/State-TalentMAP#511]
Fix pa11y errors with the enabled/disabled buttons, particularly when they appear multiple times on the same page. Consider using `shortid`.

```
Errors in http://localhost:3000/results:

 • Duplicate id attribute value "enabled-search" found on the web page.

   (#enabled-search)

   <div id="enabled-search"><button id="enabled-search-butt...</div>

 • Duplicate id attribute value "disabled-search" found on the web page.

   (#disabled-search)

   <div id="disabled-search" style="display: none;"><button class="usa-button-disab...</div>

 • This button element does not have a name available to an accessibility API. Valid names are: title attribute, element content.

   (#disabled-search-button)

   <button class="usa-button-disabled" disabled="" type="submit" id="disabled-search-button"><span
   class="usa-search-submit-...</button>
```
